web/handlers/admin/dashboard.go

```diff
@@ -17,31 +20,52 @@ type dashboardHandler struct {
 	flashes *weberrors.FlashHelper
 
 	roomState *roomstate.Manager
+	netInfo   network.ServerEndpointDetails
 	dbs       Databases
 }
 
 func (h dashboardHandler) overview(w http.ResponseWriter, req *http.Request) (interface{}, error) {
+	roomRef := h.netInfo.RoomID.Ref()
+
 	onlineRefs := h.roomState.List()
-	onlineCount := len(onlineRefs)
+
+	onlineMembers := make([]roomdb.Member, len(onlineRefs))
+	for i := range onlineRefs {
+		ref, err := refs.ParseFeedRef(onlineRefs[i])
```

Comment:
fine for now. Should probably change the `roomState.List()` to return `FeedRef`s but can happen later.

Reply:
Yeah I thought about it. I can add a TODO